### DISCUSSION TITLE
How to hide `console.log` from successful tests?

### DISCUSSION THREAD

**Discussion by chengluyu** [vitest-dev/vitest#6002] [Q&A] [answered]
I've seen this feature in one of other test frameworks. Does Vitest support this feature? If not, can I achieve the same functionality with the configuration? I checked the documentation and found a function called [onConsoleLog](https://vitest.dev/config/#onconsolelog), but there seems no way to know the source of those logs.

**Comment by J3m5** [comment, marked as answer]
We can now use the silent option with the `"passed-only"` value (https://vitest.dev/config/#silent)
